vkcs/compute/data_source_flavor.go

```diff
@@ -102,6 +103,13 @@ func DataSourceComputeFlavor() *schema.Resource {
 				Description: "The flavor visibility.",
 			},
 
+			"cpu_generation": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
There is no reason to specify `ForceNew` in data sources, this code probably was just copied from the resource's schema definition where it makes sense.

Reply:
remove from the all params?

Reply:
Up to you, it doesn't bring any misbehaviour, mentioned just to underline its pointlessness.